### ISSUE TITLE
@base64d of empty string fails with 'Error: EOF'

### ISSUE THREAD

**Issue by ksa-real** [mikefarah/yq#1209]
**Describe the bug**
See title

Version of yq: 4.25.1
Operating system: mac
Installed via: homebrew

**Command**
The command you ran:
```
echo "a: "| yq '.a|@base64d'
```
or just
```
printf ''|yq '@base64d'
```

**Actual behavior**

```
Error: EOF
```

**Expected behavior**

empty string

**Additional context**
An empty string is a correct base64 representation of an empty string and reverse.

**Comment by mikefarah** [comment]
Thanks for raising, will fix in the next release

**Comment by mikefarah** [comment]
Fixed in 4.25.2
